go/ql/lib/semmle/go/frameworks/JWT.qll

```diff
@@ -0,0 +1,91 @@
+/** Provides models of commonly used functions in common JWT packages. */
+
+import go
+
+/**
+ * Provides models of commonly used functions in common JWT packages.
+ */
+module JWT {
+  /** Gets the package name `github.com/lestrrat-go/jwx` v2. */
+  string packageLestrrat() { result = package("github.com/lestrrat-go/jwx/v2/jwt", "") }
+
+  /** Gets the package name `github.com/lestrrat-go/jwx` v1. */
+  string packageLestrratv1() { result = package("github.com/lestrrat-go/jwx/jwt", "") }
+
+  /** Gets the package name `github.com/golang-jwt/jwt` v4 and v5. */
+  string packagePathModern() {
+    result = package(["github.com/golang-jwt/jwt/v5", "github.com/golang-jwt/jwt/v4"], "")
```

Comment:
```suggestion
    result = "github.com/golang-jwt/jwt/v5", "github.com/golang-jwt/jwt/v4"]
```
No need to use the `package` predicate, since you are specifying exact versions.